### PR TITLE
pact-418

### PR DESCRIPTION
**Pull request by Ilia**

### CODE REVIEW

**Review by ilia-test — CHANGES_REQUESTED** [review]
This is a test, I am requesting changes!

File: .github/workflows/jira-issue-create.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Create Jira Ticket
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  build:
```
Comment (ilia-test) [inline .github/workflows/jira-issue-create.yml:9]: this is a test comment

File: .github/workflows/jira-issue-create.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+    name: Create Ticket
+    steps:
+    - name: Login
+      uses: atlassian/gajira-login@v3
+      env:
+        JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
```
Comment (ilia-test) [inline .github/workflows/jira-issue-create.yml:16]: this is another

**Review by ilia-test — CHANGES_REQUESTED** [review]

**Comment by Ilia** [comment]
This can go back to review!

File: .github/workflows/jira-issue-create.yml (hunk 1 above)
Comment (Ilia) [reply .github/workflows/jira-issue-create.yml:9]: test

**Comment by Ilia** [comment]
another comment!

**Comment by Ilia** [comment]
and another!

**Comment by Ilia** [comment]
this is another test

File: .github/workflows/jira-issue-create.yml (hunk 2 above)
Comment (Ilia) [reply .github/workflows/jira-issue-create.yml:16]: another test as reply

**Comment by Ilia** [comment]
test

**Comment by Ilia** [comment]
test comment in conversation tab

**Comment by Ilia** [comment]
test comment in conversation tab

**Comment by Ilia** [comment]
this is a test!

**Comment by Ilia** [comment]
this is a test comment!

File: .github/workflows/jira-issue-create.yml (hunk 1 above)
Comment (Ilia) [reply .github/workflows/jira-issue-create.yml:9]: hello!

**Comment by Ilia** [comment]
test

File: .github/workflows/jira-issue-create.yml (hunk 2 above)
Comment (Ilia) [reply .github/workflows/jira-issue-create.yml:16]: teest comment

**Comment by Ilia** [comment]
test

File: .github/workflows/jira-issue-create.yml (hunk 2 above)
Comment (Ilia) [reply .github/workflows/jira-issue-create.yml:16]: test

**Comment by Ilia** [comment]
test

**Comment by Ilia** [comment]
resolves #6

**Pull request merged**